.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
If you're using visual studio, you may want to configure your .gitignore to ignore local config files. You could use this if you want: https://github.com/github/gitignore/blob/master/VisualStudio.gitignore